db/write_batch.cc

```diff
@@ -1967,6 +1975,34 @@ Status WriteBatchInternal::InsertInto(
   return s;
 }
 
+void WriteBatchInternal::AsyncInsertInto(
+    WriteThread::Writer* writer, SequenceNumber sequence,
+    ColumnFamilySet* version_set, FlushScheduler* flush_scheduler,
+    bool ignore_missing_column_families,
+    DB* db, SafeQueue<std::function<void()>>* pool) {
+  auto write_group = writer->write_group;
+  write_group->running.fetch_add(writer->batches.size(),
+                                 std::memory_order_relaxed);
+  for (auto w : writer->batches) {
+    pool->PushBack([=]() {
+      ColumnFamilyMemTablesImpl memtables(version_set);
+      MemTableInserter inserter(sequence, &memtables, flush_scheduler,
+                                ignore_missing_column_families, 0, db, true,
```

Comment:
nit: add comment for param names whenever its not apparent, e.g. `0 /*recovering_log_number*/`